api/apiFunctions.test.tsx

```diff
@@ -1,317 +1,493 @@
-import { log } from 'console';
+import { Query } from 'appwrite';
 import {
-  addUserToLeague,
-  getAllLeagues,
-  getUserDocumentId,
+  getAllWeeklyPicks,
   loginAccount,
   logoutAccount,
   registerAccount,
+  createWeeklyPicks,
 } from './apiFunctions';
-import { account } from './config';
+import { account, databases } from './config';
+import { IUserPick } from './apiFunctions.interface';
+import { Collection } from './apiFunctions.enum';
 const apiFunctions = require('./apiFunctions');
 
 jest.mock('./apiFunctions', () => {
   const actualModule = jest.requireActual('./apiFunctions');
   return {
     ...actualModule,
-    createWeeklyPicks: jest.fn(),
     getUserWeeklyPick: jest.fn(),
-    getAllWeeklyPicks: jest.fn(),
     getCurrentUserEntries: jest.fn(),
     getAllLeagues: jest.fn(),
     getUserDocumentId: jest.fn(),
     addUserToLeague: jest.fn(),
   };
 });
 
-describe('Auth Functions', () => {
-  jest.mock('./apiFunctions', () => ({
-    loginAccount: jest.fn(),
-    registerAccount: jest.fn(),
-  }));
-
-  xdescribe('login account successful', () => {
-    it('should show user login successfully', async () => {
-      const userDummy = {
-        email: 'testemail@email.com',
-        password: 'test1234',
-      };
-      await loginAccount(userDummy);
-      expect(account.createEmailPasswordSession).toBeInstanceOf(Object);
+jest.mock('./config', () => ({
+  databases: {
+    listDocuments: jest.fn(),
```

Comment:
nit: not sorted alphabetically

Reply:
@ryandotfurrer I think I fixed this now.